plugin/src/main/kotlin/no/nav/aap/tilgang/AuthorizetionRouteConfig.kt

```diff
@@ -16,29 +16,40 @@ import tilgang.TilgangRequest
 data class AuthorizationParamPathConfig(
     val operasjon: Operasjon = Operasjon.SE,
     val avklaringsbehovKode: String? = null,
-    val approvedApplications: Set<String> = emptySet(),
+    val applicationRole: String? = null,
+    val approvedApplications: Set<String>? = null,
```

Comment:
Tenker at her bør vi ikke støtte begge deler, men kanskje holde oss til en måte?

Så stemmer egentlig for at vi fjerner application som et option, fordrer nok at vi gjør litt endringer i fakes for cc-tokens